### PR TITLE
Field docstring refactor (1/2)

### PR DESCRIPTION
**Pull request by matthew7838**
### Summary
This PR fixed half of the field folder files, there will be a second half.

### CODE REVIEW

File: RUFAS/routines/field/soil/evaporation.py [hunk 1]
```diff
@@ -219,21 +227,22 @@ def _determine_amount_water_removed(
...
+            Soil water content of given layer (mm).
         wilting_water_content : float
-            Wilting point water content of given layer (mm)
+            Wilting point water content of given layer (mm).
 
         Returns
         -------
         float
-            amount of water removed from soil layer by evaporation (mm)
+            amount of water removed from soil layer by evaporation (mm).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/evaporation.py]:
```suggestion
            Amount of water removed from soil layer by evaporation (mm).
```

File: RUFAS/routines/field/soil/layer_data.py [hunk 2]
```diff
@@ -496,31 +618,33 @@ def _initialize_carbon_pools(self, field_size: float, residue: float) -> None:
...
+        soil layer.
 
         Parameters
         ----------
             phosphorus_to_add : float
-                Amount of phosphorus to add (kg)
+                Amount of phosphorus to add (kg).
             field_size : float
-                Size of the field (ha)
+                Size of the field (ha).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/layer_data.py]: These are over-indented.

File: RUFAS/routines/field/soil/layer_data.py [hunk 3]
```diff
@@ -496,31 +618,33 @@ def _initialize_carbon_pools(self, field_size: float, residue: float) -> None:
...
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/layer_data.py]: These are over-indented.

**Review by Isgotkowitz — APPROVED** [review]
Found a couple minor things that need to be fixed, other than those LGTM.

File: RUFAS/routines/field/soil/layer_data.py (hunk 3 above)
Comment (matthew7838) [reply RUFAS/routines/field/soil/layer_data.py]: Nice catch, thanks!

File: RUFAS/routines/field/soil/evaporation.py [hunk 4]
```diff
@@ -178,23 +185,24 @@ def _determine_evaporative_demand_reduced(
...
+            Field capacity water content of given layer (mm).
         wilting_water_content : float
-            Wilting point water content of given layer (mm)
+            Wilting point water content of given layer (mm).
 
         Returns
         -------
         float
-            reduced evaporative demand for current layer based on how much water is in layer (mm)
+            reduced evaporative demand for current layer based on how much water is in layer (mm).
```
Comment (ew3361zh) [inline RUFAS/routines/field/soil/evaporation.py]:
```suggestion
            Reduced evaporative demand for current layer based on how much water is in layer (mm).
```

File: RUFAS/routines/field/soil/layer_data.py [hunk 5]
```diff
@@ -11,318 +11,439 @@
...
+        Amount of residue on the soil surface when this soil layer is initialized (kg / ha). Note: this attribute is
+        only used for initialization. After that, it cannot be used.
+    top_depth : float, optional
+        Top depth of the layer (mm).
+    bottom_depth : float, optional
+        Bottom depth of the layer (mm).
+    soil_water_concentration : float, optional, default 0.25
+        Soil water concentration of the layer (mm water / mm soil).
+    water_content : float, optional
+        Water present in the layer (mm). This field is initialized based on other parameters.
```
Comment (ew3361zh) [inline RUFAS/routines/field/soil/layer_data.py]: Is this part "This field is initialized based on other parameters." necessary? Does it need to be here?

**Review by ew3361zh — APPROVED** [review]
Really nice work, Matthew, thank you so much for taking all these on!

File: RUFAS/routines/field/soil/layer_data.py (hunk 5 above)
Comment (matthew7838) [reply RUFAS/routines/field/soil/layer_data.py]: It follows the original comments. I do think it's unnecessary. I'll remove it.

**Pull request merged**